packages/neuron-ui/src/tests/formatters/complexNumberToPureNumber/index.test.ts

```diff
@@ -1,10 +1,10 @@
-import { complexNumberToPureNumber } from 'utils/formatters'
+import { complexStringToPureNumberOrString } from 'utils/formatters'
 import fixtures from './fixtures'
 
 const fixtureTable: [string, number][] = fixtures.map(({ value, expected }) => [value, expected])
 
 describe(`Verify currency formatter`, () => {
   test.each(fixtureTable)(`%j => %s`, (value: string, expected: number) => {
-    expect(complexNumberToPureNumber(value)).toBe(expected)
+    expect(complexStringToPureNumberOrString(value)).toBe(expected)
```

Comment:
The file name should be updated according to the method name change
And more cases(`return value directly`) should be added for the update inside the method